src/clr-angular/utils/smart-popover/enums/alignment.enum.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ *
+ */
+
+/**
+ * @enum ClrAlignment
```

Comment:
`@enum` and `@description` are not valid symbols in the latest version of TSDoc. TSDoc can infer the enum automatically and the comment block defaults to the description. 
https://github.com/microsoft/tsdoc
https://microsoft.github.io/tsdoc/

Reply:
I'm going to remove these. I've since consolidated this info into the README file for hte utility.